src/components/feilvisning/FeilTabell.tsx

```diff
@@ -45,6 +45,7 @@ export const FeilTabell = ({feil , visAlleFeil, handleSetVisAlleFeil}: feilTabel
           <Ingress>Følgende feil i dokumentet må utbedres før du laster det opp på nytt:</Ingress>
             <table className="tabell tabell--stripet">
               <tbody>
+              {/* eslint-disable-next-line react/prop-types */}
```

Comment:
finnes det ikke noen måte å unngå at den reagerer på disse i det hele tatt? 🤔

Reply:
Jeg har lett etter en løsning, men fant det ikke etter en god stund med googling. Valgte denne måten for å kunne komme videre i prosessen.

Reply:
Lurt, så slipper vi masse warnings på ting som vi vet om :)